### PR TITLE
dts: nxp: nxp_imx: rt500: update pinctrl files

### PR DESCRIPTION
**Pull request by lucien-nxp**
Re-generate pinctrl files for RT500 according to data patch

### CODE REVIEW

File: dts/nxp/nxp_imx/rt/MIMXRT595SFAWC-pinctrl.h [hunk 1]
```diff
@@ -1,8 +1,8 @@
 /*
- * NOTE: File generated by lpc_cfg_utils.py
+ * NOTE: File generated by gen_soc_headers.py
  * from MIMXRT595SFAWC/signal_configuration.xml
  *
- * Copyright (c) 2022, NXP
+ * Copyright (c) 2024 NXP
```
Comment (danieldegrasse) [inline dts/nxp/nxp_imx/rt/MIMXRT595SFAWC-pinctrl.h:5]: I know the script (unfortunately) doesn't handle this, but we need to make these copyrights distinct, like so:
```suggestion
 * Copyright 2022, 2024 NXP
```

Comment (lucien-nxp) [reply dts/nxp/nxp_imx/rt/MIMXRT595SFAWC-pinctrl.h]: OK, Updated.

**Comment by lucien-nxp** [comment]
Pinout need to be double check. So this PR need to wait the result of confirmation.

File: dts/nxp/nxp_imx/rt/MIMXRT595SFAWC-pinctrl.h [hunk 2]
```diff
@@ -1,8 +1,8 @@
 /*
- * NOTE: File generated by lpc_cfg_utils.py
+ * NOTE: File generated by gen_soc_headers.py
  * from MIMXRT595SFAWC/signal_configuration.xml
  *
- * Copyright (c) 2022, NXP
+ * Copyright (c) 2022, 2024 NXP
```
Comment (dleach02) [inline dts/nxp/nxp_imx/rt/MIMXRT595SFAWC-pinctrl.h:5]: Please drop the (c)

File: dts/nxp/nxp_imx/rt/MIMXRT595SFFOC-pinctrl.h [hunk 3]
```diff
@@ -1,8 +1,8 @@
 /*
- * NOTE: File generated by lpc_cfg_utils.py
+ * NOTE: File generated by gen_soc_headers.py
  * from MIMXRT595SFFOC/signal_configuration.xml
  *
- * Copyright (c) 2022, NXP
+ * Copyright (c) 2022, 2024 NXP
```
Comment (dleach02) [inline dts/nxp/nxp_imx/rt/MIMXRT595SFFOC-pinctrl.h:5]: Drop (c)

**Comment by dleach02** [comment]
@lucien-nxp We need a Zephyr side PR to ensure CI is okay

**Comment by danieldegrasse** [comment]
Actually looking at this more, this is strictly additions- @dleach02 do you think we really need a full CI run for this change?

**Comment by danieldegrasse** [comment]
> This is always needed. It's a Zephyr project policy, not an NXP policy.

Yeah, the documented process does request a Zephyr PR always be created. @lucien-nxp can you please make one?

**Pull request closed without merging**